### DISCUSSION TITLE
Starting new rows, that impact corrected indentation for formatting

### DISCUSSION THREAD

**Discussion by webbgroup** [PHP-CS-Fixer/PHP-CS-Fixer#6816] [Q&A]
## How do I get rid of this type of behavior, where the phpfixer formatting changes not only the new line, but pulls the line to the beginning of the page. This ends up impacting the collapsing of the code:
```
-                                <input type="radio" name="example" class="rating" value="1" <?php if (round($score)==1) { echo 'checked'; }?>/>
+                                <input type="radio" name="example" class="rating" value="1" <?php if (1 == round($score)) {
+  echo 'checked';
+}?>/>
```
This is what is being run:
```
(braces, yoda_style, binary_operator_spaces)
```

**Comment by Wirone** [comment]
Please have in mind that Fixer is focused on modern PHP, that does not mix code with HTML markup (see [this comment](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/6788#issuecomment-1438688702)), that's why I'm going to close this discussion 🙂.

**Reply by webbgroup** [reply]
Yes but that doesn't explain how to resolve the issue. And yes, we are using modern PHP.

**Reply by Wirone** [reply]
We don't provide support for PHP files that mix HTML code. You can apply rules there, but we don't provide fixes for HTML-related cases 🙂.
